Add xeon_cogs to unapproved list

Avexiis opened the pull request:
Cog just tracks vouches in specified channels.

Flame442 commented:
You should not be calling `bot.tree.sync` every time the cog loads, as that is a heavily rate limited endpoint. If every 3rd party cog attempted to call that when it loaded, most would fail and would be rejected in the loading process. It also won't actually cause your commands to appear until they are `[p]slash enable`d, negating any potential benefit. This needs to be fixed before I can add your repo to the index.

Additionally,
- You should consider using [`@commands.hybrid_command`](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.hybrid_command) if you want commands to be available as both text and slash commands.
- You are currently missing a repo level [`info.json`](https://docs.discord.red/en/stable/guide_publish_cogs.html) file.
- You should consider using a member scoped config value instead of using a guild scoped config value `vouch_data` and manually storing user ids.

Avexiis commented:
> You should not be calling `bot.tree.sync` every time the cog loads, as that is a heavily rate limited endpoint. If every 3rd party cog attempted to call that when it loaded, most would fail and would be rejected in the loading process. It also won't actually cause your commands to appear until they are `[p]slash enable`d, negating any potential benefit. This needs to be fixed before I can add your repo to the index.
> 
> Additionally,
> 
> * You should consider using [`@commands.hybrid_command`](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.hybrid_command) if you want commands to be available as both text and slash commands.
> * You are currently missing a repo level [`info.json`](https://docs.discord.red/en/stable/guide_publish_cogs.html) file.
> * You should consider using a member scoped config value instead of using a guild scoped config value `vouch_data` and manually storing user ids.

Makes sense. I've removed bot.tree.sync, added info.json to the repo level, & taken you up on your suggestions.
